### PR TITLE
Add Yellow to Getting started section

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Add Home Assistant Yellow as an option in the Getting started section.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/installation/yellow.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        </div>
+        <div style="margin-top: 72px;">
+            <div>
+                <h1>Installation instructions</h1>
+            </div>
+            <div class="material-card text" style="margin-bottom: 72px;">
+                <div class="content-container">
+                    <div>
+                        <p>Already have the hardware?</p>
+                        <p>For installation instructions, checkout the <a
```
Comment (mrand) [inline source/installation/yellow.html]:
```suggestion
                        <p>For installation instructions, check out the <a
```

Comment (c0ffeeca7) [reply source/installation/yellow.html]: Thank you for the catch, @mrand !

File: source/installation/yellow.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    <div>
+                        <img src="/images/installation/yellow-logo.png" style="margin-bottom: 24px;">
+                    </div>
+                    <div>
+                        <p>Our take on Home Automation.</p>
+                        <p><img src="/images/installation/home-assistant-yellow-exploded-and-labeled.png"
+                                style="margin-bottom: 16px;"> </a>
+                        </p>
+                        <p>Don't have the hardware yet? </br>
+                            Visit the <a href="https://www.home-assistant.io/yellow">Home Assistant Yellow product page</p>
```
Comment (frenck) [inline source/installation/yellow.html]:
```suggestion
                            Visit the <a href="/yellow">Home Assistant Yellow product page</p>
```

File: source/installation/yellow.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <div style="margin-top: 72px;">
+            <div>
+                <h1>Installation instructions</h1>
+            </div>
+            <div class="material-card text" style="margin-bottom: 72px;">
+                <div class="content-container">
+                    <div>
+                        <p>Already have the hardware?</p>
+                        <p>For installation instructions, check out the <a
+                                href="https://www.home-assistant.io/yellow">Home Assistant Yellow documentation</a></p>
```
Comment (frenck) [inline source/installation/yellow.html]:
```suggestion
                                href="/yellow">Home Assistant Yellow documentation</a></p>
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**